spec/calendar.html

```diff
@@ -561,11 +579,14 @@ <h1>
           _year_: an integer,
           _month_: an integer,
           _day_: an integer,
-        ): an integer
+        ): a Record with fields [[Week]] (a positive integer) and [[Year]] (an integer)
```

Comment:
could we name this record type, like "a WeekYear Record", so there's a dfn to link to?

Reply:
For me personally this seems to give too much emphasis to a record that's just used in two places, given that there are relatively few records with their own type definition in ECMA-262 and they are fundamental things like the completion record. But I've added it on in a separate commit, we can see what people say about it.